Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -59,6 +61,9 @@ codeunit 4022 "GP Vendor Migrator"
         if RecordIdToMigrate.TableNo() <> Database::"GP Vendor" then
             exit;
 
+        if GPCompanyAdditionalSettings.GetMigrateOnlyPayablesMaster() then
```

Comment:
should this be in OnMigrateVendorPostingGroups too?

Reply:
Added